app/src/main/java/com/garan/wearwind/screens/Connect.kt

```diff
@@ -99,7 +99,9 @@ fun ConnectScreen(
     ConstraintLayout(constraintSet = constraintSet, modifier = Modifier.fillMaxSize()) {
```

Comment:
What is keeping you on ConstraintLayout here?

Reply:
Haha!! Nothing! It's literally next on my todo list!!